### PR TITLE
GitHub collaboration resources

### PR DESCRIPTION
**Pull request by quadrophobiac**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Of the github integrations we use two will be utilised to check appraise your contribution. In order of priority these are
+    - Travis ensures that all tests (existing and additions) pass
+    - Travis/Coveralls ensures that overall test coverage for lines of code meets a certain threshold. If this metric dips below what it previously was for the repository you’re pushing to then your PR will be rejected
+    - Gemnasium ensures dependencies are up to date
+- Once your PR is published and passes the above checks a repository administrator will review your contribution. Where appropriate comments may be provided and amendments suggested before your PR is merged into Master.
+- Once your PR is accepted you will be granted push access to the repository you have contributed to! Congratulations on joining our community, you’ll no longer need to work from forks.
+
+If you make a contribution to another repository in the Toolbox you will be expected to repeat this process. Read more about that [here](https://github.com/theodi/toolbox/blob/master/README.md#push-access).
+
+## Code Style Guide
```
Comment (Stephen-Gates) [inline .github/CONTRIBUTING.md]: Ruby coding style not relevant for comma chameleon

Comment (quadrophobiac) [inline .github/CONTRIBUTING.md]: Good spot - and reminds me of a memo I had for just that issue

**Comment by chris48s** [comment]
At the moment, the linting rules that JSHint is currently enforcing don't 'agree' with the code style guide linked.

As an example, JSHint is configured to require semicolons (this is the default for JSHint, but you can [turn it off](http://jshint.com/docs/options/#asi) ), whereas the 'standard js' coding style says you should not use semicolons and rely on ASI.

As another example, JSHint currently requires strict comparisons in all situations (again, the default for JSHint) but the proposed conventions allow `obj == null` (again this is [configurable](http://jshint.com/docs/options/#eqnull) ).

I'm aware that I set up the JSHint config (I mainly just went with the defaults), but I don't raise this because I want to start a big debate about semicolons. GitHub has [enough of those already](https://github.com/twbs/bootstrap/issues/3057) ;) Being an electron app, it is probably better to move into line with the electron community conventions - I just think that part of adopting style guidelines should be updating the existing codebase and linting rules (or [tools](https://www.npmjs.com/package/standard) ) to meet/enforce those guidelines.

**Comment by pezholio** [comment]
I think Standard.js would be a good shout for this 👍

**Comment by Stephen-Gates** [comment]
Standard.js seems perfect given the [list of users](https://standardjs.com/index.html#who-uses-javascript-standard-style) includes Electron, NPM, GOV.UK and more.

**Comment by chris48s** [comment]
So I think the steps to adopting that should be:

- [ ] Add [standard js](https://www.npmjs.com/package/standard) to project requirements
- [ ] Bring existing code into line with guidance:
  * Run `standard --fix` on codebase to automatically convert as much as possible to to standard js conventions
  * Mop any issues that need manual intervention

- [ ] Remove JsHint config files (`.jshintrc`, `.jshintignore`) from root/subdirs + JsHint package
- [ ] Edit Travis config to run Standard Js linter in CI build instead of JsHint

**Comment by quadrophobiac** [comment]
Hey everyone - thanks so much for the input on this, really great discussion and especially thanks to you @chris48s for speccing out what should be done. I'll get those into tickets and into the backlog!

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Contributing to Comma Chameleon
+
+Open Data Certificates is open source, and contributions are gratefully accepted!
```
Comment (Stephen-Gates) [inline .github/CONTRIBUTING.md]: Open Data Certificates should be Comma Chameleon

**Pull request merged**